container/isotovideo/Dockerfile.qemu-kvm

```diff
@@ -1,5 +1,6 @@
 #!BuildTag: isotovideo:qemu-kvm
 
 FROM opensuse/tumbleweed
-RUN zypper -n in os-autoinst-qemu-kvm && zypper clean
+# Provide "jq" as convenience to work with the os-autoinst output
+RUN zypper -n in os-autoinst-qemu-kvm jq && zypper clean
```

Comment:
To be honest this seems pretty arbitrary. The clean approach would be to use a dedicated image which includes all tools needed **or** even make it a proper os-autoinst feature if it caters to a common use case.

Reply:
ok, updated to use two new explicit container images instead.

Reply:
And I've already wanted to comment why not just putting the tool into the image we already have. I'm not sure what's best but creating a distinct container for each set of tools we ever need seems not like a good idea. I guess since it is just one extra container for now it is ok.